### ISSUE TITLE
Comment on prosody ports

### ISSUE THREAD

**Issue by 0x64746b** [alexh-name/0101010.one#7]
I think it'd be nice if the guide mentioned that the [values for the ports](https://github.com/alexh-name/0101010.one/blob/master/src/hosting/Uberspace/prosody.txt#L175) are placeholders that need to be replaced :) A link to the uberspace documentation on how [the firewall selects ports for you](https://wiki.uberspace.de/system:ports#firewall) would be great!

**Comment by alexh-name** [comment]
Good point. I will mention it and link to the wiki page. The original guide mentioned getting ports assigned manually how it used to be when it still was a thing at uberspace.

**Comment by 0x64746b** [comment]
Hmpf, I completely read over your comment wrt to the ports. Multiple times *d'oh* Link is still nice. Thx!
